public/locales/pl/gamepage.json

```diff
@@ -121,36 +121,37 @@
       "stop": "Gra włączona (Zatrzymaj)"
     },
     "saves": {
-      "syncing": "Synchronizacja zapisów gry"
-    "submenu": {
+      "syncing": "Synchronizacja zapisów gry",
+      "submenu": {
```

Comment:
pretty sure @flavioislima has fixed issues with the Polish translations, maybe a merge from the main branch will fix issues with these translations.